.github/workflows/ci.yml

```diff
@@ -54,10 +67,12 @@ jobs:
     - uses: actions/checkout@v2
       with:
         repository: wildfly/wildfly
+        ref: ${{ github.base_ref }}
```

Comment:
One thing to note is this will require branches to exactly match. e.g. `main`, `28.x`. Each project would be required to have these branches.

Reply:
yes, that's intended, BOMs and QS always follow same branch names as WildFly